Source/Scene/OrthographicFrustum.js

```diff
@@ -23,13 +17,13 @@ define([
      * define the unit vector normal to the plane, and the w component is the distance of the
      * plane from the origin/camera position.
      *
-     * @alias OrthographicFrustum
+     * @alias OrthographicOffCenterFrustum
```

Comment:
For CHANGES.md, do we have any breaking changes?  If so, we can deprecate them over **one** release since this part of the API is not likely widely used.